### PR TITLE
add optimism, arbitrum, and avalanche

### PR DESCRIPTION
**Pull request by auryn-macmillan**
This PR adds optimism, arbitrum, and avalanche

### CODE REVIEW

File: packages/exit-app/src/utils/networks.ts [hunk 1]
```diff
@@ -40,6 +50,10 @@ export const NETWORK_DEFAULT_RPC: Record<NETWORK, string> = {
   [NETWORK.BSC]: 'https://bsc-dataseed.binance.org',
   [NETWORK.XDAI]: 'https://rpc.gnosischain.com',
   [NETWORK.POLYGON]: 'https://polygon-rpc.com',
+  [NETWORK.XDAI]: 'https://rpc.gnosischain.com',
```
Comment (jfschwarz) [inline packages/exit-app/src/utils/networks.ts:53]: This is duplicated now

```suggestion
```

**Review by jfschwarz — CHANGES_REQUESTED** [review]
These keys still have to be added as env vars to the workflow here: https://github.com/gnosis/zodiac-module-exit/blob/master/.github/workflows/prod-release-deploy.yaml#L7

However, I would suggest to just get rid of all these env vars and inline the keys in directly in explorers.ts. There's no point in hiding them, they will be part of the public JS app bundle anyways.

File: packages/exit-app/src/utils/networks.ts (hunk 1 above)
Comment (auryn-macmillan) [reply packages/exit-app/src/utils/networks.ts:53]: Good catch. Removing.

**Comment by auryn-macmillan** [comment]
> However, I would suggest to just get rid of all these env vars and inline the keys in directly in explorers.ts. There's no point in hiding them, they will be part of the public JS app bundle anyways.

Yeah, agreed. However, for the sake of ease now, I just added them to the workflow.

**Review by jfschwarz — APPROVED** [review]
👌

**Pull request merged**